src/demos/chip-group.html

```diff
@@ -0,0 +1,506 @@
+      }
+
+      .themed-children calcite-chip:nth-of-type(2) {
+        --calcite-ui-brand: var(--calcite-ui-success);
+        --calcite-ui-text-1: var(--calcite-ui-success);
+        --calcite-ui-border-1: var(--calcite-ui-success);
+        --calcite-ui-foreground-1: rgba(0, 128, 0, 0.15);
+      }
+
+      .themed-children calcite-chip:nth-of-type(3) {
+        --calcite-ui-brand: var(--calcite-ui-info);
+        --calcite-ui-text-1: var(--calcite-ui-info);
+        --calcite-ui-border-1: var(--calcite-ui-info);
+        --calcite-ui-foreground-1: rgba(0, 0, 255, 0.15);
+      }
+
+      .themed-children calcite-chip:nth-of-type(4) {
+        --calcite-ui-brand: orange;
+        --calcite-ui-text-1: orange;
+        --calcite-ui-border-1: orange;
+        --calcite-ui-foreground-1: rgba(255, 165, 0, 0.15);
+      }
+
+      .themed-children calcite-chip:nth-of-type(5) {
+        --calcite-ui-brand: purple;
+        --calcite-ui-text-1: purple;
+        --calcite-ui-border-1: purple;
+        --calcite-ui-foreground-1: rgba(128, 0, 128, 0.15);
+      }
+
+      .themed-children calcite-chip:nth-of-type(6) {
+        --calcite-ui-brand: pink;
+        --calcite-ui-text-1: pink;
+        --calcite-ui-foreground-1: rgba(255, 192, 203, 0.15);
+      }
+
+      .themed-children calcite-chip:nth-of-type(7) {
+        --calcite-ui-brand: teal;
+        --calcite-ui-text-1: teal;
+        --calcite-ui-foreground-1: rgba(0, 128, 128, 0.15);
+      }
+
+      .themed-children-card calcite-chip:first-of-type {
+        --calcite-ui-foreground-2: rgb(222 239 220);
+        --calcite-ui-text-1: var(--calcite-ui-success);
+        --calcite-ui-icon-color: var(--calcite-ui-success);
+      }
+
+      .themed-children-card calcite-chip:nth-of-type(2) {
+        --calcite-ui-foreground-2: rgb(221 238 249);
+        --calcite-ui-text-1: var(--calcite-ui-info);
+        --calcite-ui-icon-color: var(--calcite-ui-info);
+      }
+
+      .themed-children-card calcite-chip:nth-of-type(3) {
+        --calcite-ui-foreground-2: rgb(221 238 249);
+        --calcite-ui-text-1: var(--calcite-ui-info);
+        --calcite-ui-icon-color: var(--calcite-ui-info);
+      }
+
+      hr {
+        margin: 60px 0;
+        border-top: 1px solid var(--calcite-ui-border-2);
+      }
+
+      h2 {
+        margin: 80px auto 40px;
+        font-size: 22px;
+        text-align: center;
+        max-width: 50%;
+        font-weight: 500;
+      }
+    </style>
+    <script src="_assets/head.js"></script>
+  </head>
+
+  <body>
+    <demo-dom-swapper>
+      <!-- Header -->
+      <div class="child">
+        <h2>Chip group and selection mode examples</h2>
+      </div>
+      <div class="child">
+        <h2>Examples - selection modes with <code>chip-group</code> and <code>chip selectable</code></h2>
+      </div>
+      <!-- Neutral (default) -->
+      <div class="parent">
+        <div class="child">None</div>
+        <calcite-chip-group>
```

Comment:
If the `label` prop is in line with consistency across our component, we would need to update all the `chip-group`'s in the demo to include a `label` prop, such as "None" in this case.

Reply:
Added :)